### PR TITLE
Helper functions

### PR DESCRIPTION
**Pull request by franksrevenge**
Howdy,

Here are a couple of helpers that I think would be very helpful with MrHorse.

* Define a set of policies of which *at least one* must be satisfied: `MrHorse.orPolicy('policyName1', 'policyName2', ...)`
* Load multiple policies from a single file: `module.exports = { policyName1 : function() { ... }, policyName2 : ... };`
* Add policies programmatically: `server.plugins.mrhorse.addPolicy(policyName, policy)`
* Test whether policy exists: `server.plugins.mrhorse.hasPolicy(policyName)`

As far as I can tell, this is fully backward compatible and has 100% test coverage.

### CODE REVIEW

**Comment by mark-bradshaw** [comment]
@franksrevenge Thanks.  I'll take a look.

File: test/index.js [hunk 1]
```diff
@@ -732,4 +825,53 @@ lab.experiment('Normal setup', function (done) {
         });
     });
 
+    lab.test('policy can be added programmatically', function (done) {
+        server.plugins.mrhorse.addPolicy('yetAnotherPolicy', (request, reply, callback) => callback(null, true));
```
Comment (mark-bradshaw) [inline test/index.js]: Please add a negative assertion first to make sure the 'yetAnotherPolicy' policy doesn't exist at first.

File: test/index.js [hunk 2]
```diff
@@ -732,4 +825,53 @@ lab.experiment('Normal setup', function (done) {
...
+
+        Code.expect(server.plugins.mrhorse.hasPolicy('yetAnotherPolicy')).to.equal(true);
+        done();
+    });
+
+    lab.test('hasPolicy returns false on non-existent policies', function (done) {
+        Code.expect(server.plugins.mrhorse.hasPolicy('thisPolicyAbsolutelyDoesNotExist')).to.equal(false);
+        done();
+    });
```
Comment (mark-bradshaw) [inline test/index.js]: I'd also like to see a test where the programatically added policy is attached to a route and tested.

**Comment by franksrevenge** [comment]
Should be all fixed now.

**Comment by mark-bradshaw** [comment]
Thanks for the PR.  Good functionality here.  It's merged and I'll be releasing soon.

**Pull request merged**